lib/sprockets/sass/v3/functions.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+module Sprockets
+  module Sass
+    module V3
+      # Module used to inject helpers into SASS engine
+      module Functions
```

Comment:
Currently there are no changes, but i copied it because i wanted to ask about compatibility with sprockets-helpers.  If we need to maintain that compatibility, i have to refactor this functions. 

If not, i can just delete it because the method ```get_class_by_version``` will automatically pick the highest version available.